Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -379,6 +408,14 @@ void MainWindow::ShowAboutDialog()
   about->show();
 }
 
+void MainWindow::ShowHotkeyDialog()
+{
+  m_hotkey_window->ChangeMappingType(MappingWindow::Type::MAPPING_HOTKEYS);
+  m_hotkey_window->show();
```

Comment:
Ah, you're right: http://doc.qt.io/qt-4.8/qt-performance.html#programming-style

`exec` still seems like the way to go though, right?